search.go

```diff
@@ -275,6 +276,7 @@ type SearchRequest struct {
 	Sort             search.SortOrder  `json:"sort"`
 	IncludeLocations bool              `json:"includeLocations"`
 	Score            string            `json:"score,omitempty"`
+	SearchAfter      []string          `json:"search_after"`
```

Comment:
snake_case -> search_after when all existing are camelCase?

Reply:
good_catch!

Reply:
Actually `snake_case` was what we intended to use for all JSON, and `camelCase` in the Go code.

Here are some longer-lived multi-word names:

https://github.com/blevesearch/bleve/blob/master/search.go#L447-L448
https://github.com/blevesearch/bleve/blob/master/search.go#L123-L124

I think we just missed the naming issue when we added `includeLocations` to the request.